lib/screens/start_screen.dart

```diff
@@ -176,26 +176,25 @@ class DoOnboardWidget extends StatelessWidget {
   Widget build(BuildContext context) {
     return ElevatedButton(
       onPressed: () async {
+        AtOnboardingConfig atOnboardingConfig = AtOnboardingConfig(
+          atClientPreference: await loadAtClientPreference(),
+          rootEnvironment: AtEnv.rootEnvironment,
+          domain: AtEnv.rootDomain,
+          appAPIKey: AtEnv.appApiKey,
+        );
         AtOnboardingResult onboardingResult = await AtOnboarding.onboard(
           context: context,
-          config: AtOnboardingConfig(
-            atClientPreference: await loadAtClientPreference(),
-            rootEnvironment: AtEnv.rootEnvironment,
-            domain: AtEnv.rootDomain,
-            appAPIKey: AtEnv.appApiKey,
-          ),
+          config: atOnboardingConfig,
           isSwitchingAtsign: true,
         );
         switch (onboardingResult.status) {
           case AtOnboardingResultStatus.success:
-            Navigator.push(
-                context,
-                MaterialPageRoute(
-                    builder: (_) => const StartScreen(
-                        // futurePreference: this.futurePreference,
-                        )));
+            AtOnboarding.reset(context: context, config: atOnboardingConfig);
```

Comment:
@nickelskevin AtOnboarding.reset reset might not work as expected.
`AtOnboarding.reset` gives an screen where user can select atSigns they want to remove from keychain.

Just after reset screen is shown, we are pushing a new screen, which will hide reset screen and user might not notice it.
 `  Navigator.push(context,
                MaterialPageRoute(builder: (_) => const StartScreen()));`